restws.install

```diff
@@ -6,16 +6,28 @@
  */
 
 /**
- * Convert variables to config
+ * Clear the page cache to throw out any RESTWS responses that could be
+ * delivered on HTML requests.
  */
-function restws_update_1000() {
-  // Migrate variables to config.
-  $config = config('restws.settings');
-  $config->set('restws_query_max_limit', update_variable_get('restws_query_max_limit', 100));
-  $config->set('restws_debug_log', update_variable_get('restws_debug_log', ''));
-  $config->save();
+function restws_update_7001() {
```

Comment:
This should be renamed `restws_update_1001()` in order to indicate this is a Backdrop update hook.